WordPress/Classes/ViewRelated/Blaze Campaigns/BlazeCampaignsViewController.swift

```diff
@@ -26,13 +34,39 @@ final class BlazeCampaignsViewController: UIViewController, NoResultsViewHost, B
         return tableView
     }()
 
+    private lazy var promoteButton: UIButton = {
+        let button = UIButton(type: .system)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.addTarget(self, action: #selector(buttonCreateCampaignTapped), for: .touchUpInside)
+
+        var configuration = UIButton.Configuration.filled()
+        configuration.title = Strings.promoteButtonTitle
+        configuration.titleTextAttributesTransformer = UIConfigurationTextAttributesTransformer({ incoming in
+            var outgoing = incoming
+            outgoing.font = WPStyleGuide.fontForTextStyle(.body, fontWeight: .semibold)
+            return outgoing
+        })
+        configuration.image = UIImage(systemName: "plus", withConfiguration: UIImage.SymbolConfiguration(scale: .large))
+        configuration.imagePlacement = .leading
+        configuration.imagePadding = 8
+        configuration.baseBackgroundColor = UIColor(
```

Comment:
Nit: visual effect view will look good here.

Reply:
I've started a discussion here in this Slack thread! p1688729817258079-slack-C04LJFS1G5P

If we do decide to use visual effect view, I'll open a different PR. I'd have to move the button out of a stackview and pin it to the rootview instead to get the blur effect. And also change the animation logic since I'm currently relying on the stackview isHidden animation we get for free